### DISCUSSION TITLE
Cask formulae unavailable

### DISCUSSION THREAD

**Discussion by mcooperman** [Homebrew/discussions#4234] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 17c872fb5275d87922a56416587cb439a5064354
Last commit: 2 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: c07a17eb071cc8319e9c507c135daa17953ecbd7
Core tap last commit: 73 minutes ago
Core tap branch: master
Core tap JSON: 18 Feb 18:24 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
no_proxy: 127.0.0.1,localhost,*.local, <REDACTED>
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

brew update fails on this:

==> Downloading https://formulae.brew.sh/api/cask.json
curl: (22) The requested URL returned error: 404

Error: Failure while executing; `/usr/bin/env /opt/homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.0.1\ \(Macintosh\;\ arm64\ Mac\ OS\ X\ 12.6.1\)\ curl/7.79.1 --header Accept-Language:\ en --fail --progress-bar --location --remote-time --output /Users/coopermm/Library/Caches/Homebrew/api/cask.json --compressed --speed-limit 100 --speed-time 5 --progress-bar https://formulae.brew.sh/api/cask.json` exited with 22. Here's the output:
curl: (22) The requested URL returned error: 404

trying the URL through a browser also returns 404 Error

### Relevant casks

any/all

**Comment by p-linnane** [comment]
I'm not able to reproduce this. This page is served from GitHub Pages. Can you clear your cache and try again?

**Reply by mcooperman** [reply]
whatever it was, it's gone. did not even need to clear cache. must have been a glitch. thx
